docs/source/technical_tutorials/access/lpdaac_gedi_access.ipynb

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
<ol><li>Jamison, your name is missing from the Author list.</li><li>We need a big banner at the top about this being a temporary work around, as the S3 version isn't officially released yet.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>